src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -30,7 +31,8 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>$(OutputPath)\Md2Html\</OutputPath>
+    <SharedOutputPath>$(OutputPath)\Md2Html\</SharedOutputPath>
+    <OutputPath>bin\AnyCPU\Release\</OutputPath>
```

Comment:
can you explain this and why it needs two output paths and hardcoded path?

Reply:
`SharedOutputPath` points to a `md2Html` directory in our shared bin folder.
`OutputPath` points to a local output folder relative to the project file.

`ILMerge` is then used for building a single .exe that is placed in the shared folder from the files in the local bin folder.